src/main/java/com/deque/axe/android/AxeView.java

```diff
@@ -132,7 +132,7 @@ private AxeView(
 
     // This should be the last thing we do in case we decide parent/children relationships
     // contribute to ID calculation.
-    this.axeViewId = this.hashCode();
+    this.axeViewId = Integer.toHexString(this.hashCode());
```

Comment:
What does `toHexString` do? Does it convert `axeViewId` (Int) to be in the `0x********....` format?  If so, is this a requirement for iOS to do as well?

Reply:
Not a requirement for iOS. A `string` is a `string`. I decided to make it a HEX String for the java library, as the long term plan will be to create a longer string in the future, with lower odds of hash collisions.